pkg/databind/internal/secrets/cyberarkapi_test.go

```diff
@@ -0,0 +1,100 @@
+package secrets
+
+import (
+	"fmt"
+	"github.com/newrelic/infrastructure-agent/pkg/databind/pkg/data"
+	. "net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+)
+
+func TestCyberArkAPI(t *testing.T) {
+
+	ts := httptest.NewServer(HandlerFunc(func(w ResponseWriter, r *Request) {
```

Comment:
Why don't we use `newHttpTestServer()` here? We can reuse it if we will add response writer there.

Reply:
It's not clear what you mean, what would the refactoring look like?

Reply:
I think it may be something like this.
```
func newHttpTestServer(response string, rc int) *httptest.Server {
	return httptest.NewServer(HandlerFunc(func(w ResponseWriter, r *Request) {
		w.WriteHeader(rc)
                 w.Write([]byte(response))
	}))
}
```

and then you can use it like 
```
ts := newHttpTestServer(`{
  "Content": "CI<s0H9>k2zUPXM",
  "PolicyID": "ALL-APPX-WIN-SL-XNCVRRI-DEFAULT",
  "SequenceID": "2",
  "UserName": "testuser",
  "CPMStatus": "success",
  "Folder": "Root",
  "Safe": "ALL-NERE-WIN-A-NEWRELIC-UP",
  "Address": "localhost",
  "Name": "ALL-localhost-testuser",
  "LogonDomain": "localhost",
  "DeviceType": "Operating System",
  "LastTask": "ChangeTask",
  "RetriesCount": "-1",
  "LastSuccessChange": "1589288852",
  "CreationMethod": "PVWA",
  "PasswordChangeInProcess": "False"
}`, 200)
```

I didn't check this in IDE, may have syntax errors.

Reply:
Ah, I see what you're thinking.  Thanks.

Reply:
Changed